### ISSUE TITLE
Line mapping in source maps don't seem to be picked up when using JetBrains WebStorm

### ISSUE THREAD

**Issue by pbadenski** [swc-project/swc#2008]
This is quite difficult for me to debug - I'm sorry in advance - this issue is not reported as clearly as I would like it to be.

I'm trying to debug TypeScript codebase with JetBrains WebStorm when using swc and line numbers aren't showing correct.

It looks like some information from source maps is coming through - as the file names show as ".ts". However the line numbers seems to be from the generated JS file rather than original TS file - so I'm guessing line mapping information doesn't get read properly. I'm not sure why that would be... It works with babel. I tried to compare with babel source map, but they're quite different. One thing that I noticed with babel source maps is they nearly always have an entry for `generatedColumn: 0` which I think might be used by a debugger:
```
{
  source: '/Users/pbadenski/workspace/..../eitherFormulaOrNormalEditor.spec.ts',
  generatedLine: 52,
  generatedColumn: 0,
  originalLine: 45,
  originalColumn: 4,
  name: 'editor'
}
```

This is just a hunch though.

Please let me know if you can think of a better way I can troubleshoot/help with this. I'm really keen on figuring this out

**Version**
The version of @swc/core: 1.2.72

**Comment by pbadenski** [comment]
Just wondering if there's any way I can move this forward. But also - do people seem to experience similar with other editors (eg. VS Code?)

**Comment by pbadenski** [comment]
Looks that it's fixed in 1.2.84 🎉 

THANK YOU!!!
